components/layout/CommunityDocsLayout.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import CommunityDocsNav from '../navigation/CommunityDocsNav';
+import CommunityLayout from './CommunityLayout'
+
+function CommunityDocsLayout({
+  children,
+  post,
+  navItems
+}) {
+  return (
+    <CommunityLayout
+      title="Community docs and resources"
+      description="Getting started with docs"
```

Comment:
```suggestion
      description="Getting started with community docs"
```

I think we need to more clearly differentiate that these are a separate set of docs than the spec docs